src/parser/index.js

```diff
@@ -111,46 +149,212 @@ const addOptionToScript = ({ script, yamlFile, yamlFileName, key }) => {
+ * @param {Script} param.script - simple script to a command to
+ * @param {object} param.file   - file to parse the command from
+ * @param {string} param.key    - key to use to parse the command from the file
+ */
+const addCommandToSimpleScript = ({ script, file, key }) => {
+  const re = new RegExp(SIMPLE_SCRIPT_OPTION_SEPARATOR, 'g');
+  const commandKey = `${script.getName()}.${key.replace(re, KEY_SEPARATOR)}`;
+  const directive = file[key];
+  const directives = [directive];
+  const command = new Command({ directives });
+  script.addCommand({ commandKey, command });
+};
+
+/**
+ * Parse a simple script
+ *
+ * @param {object} param             - object parameter
+ * @param {Script} param.script      - the simple script parsed from the file
+ * @param {object|string} param.file - the file to parse the simple script from
+ *
+ * @returns {Script} script    - the simple script parsed from the file
+ */
+const parseSimpleScript = ({ script, file }) => {
+  if (isString(file)) {
+    const command = new Command({ directives: [file] });
+    script.addCommand({ commandKey: script.getName(), command });
+    return script;
   }
+
+  const keys = Object.keys(file);
+
+  // Add top level option to script
+  addOrUpdateOptionToSimpleScript({
+    script,
+    optionKey: script.getName(),
+    choices: [...getChoicesFromKeys({ keys }), QUIT_COMMAND],
+  });
+
+  keys.forEach(key => {
+    // Add command
+    addCommandToSimpleScript({ script, file, key });
+
+    // Add options
+    const optionKeys = getOptionsKeysFromKey(key);
+    optionKeys.forEach(optionKey => {
+      // Add option
+      const choices = getChoicesFromKeys({ keys, startingKey: optionKey });
+      addOrUpdateOptionToSimpleScript({
+        script,
+        optionKey: `${script.getName()}.${optionKey}`,
+        choices: [...choices, BACK_COMMAND, QUIT_COMMAND],
+      });
+    });
+  });
+  return script;
 };
 
 class Parser {
   /**
-   * Parse a yaml file into commands, options, messages and directories
+   * Parse a yaml file into commands, options, messages, variables and directories
+   *
+   * @param {object} param                 - object parameter
+   * @param {string} param.fileName        - the name of the yaml file to be loaded and parsed
+   * @param {ScriptTypes} param.scriptType - the type of script to parse (simple or advanced)
    *
-   * @param Object yamlFileName - the name of the yaml file to be loaded and parsed
+   * @returns {Script} script        - the script loaded into memory
    */
-  static getScript(yamlFileName) {
-    if (!isValidYamlFileName(yamlFileName)) {
-      printMessage(formatMessage(globalMessages.invalidYamlFile, { yamlFileName }));
+  static getScriptFromYamlFile({ fileName, scriptType }) {
+    if (!isValidYamlFileName(fileName)) {
+      printMessage(formatMessage(globalMessages.invalidYamlFile, { fileName }));
       forceExit();
     }
-    const yamlFile = loadYamlFile(yamlFileName);
```

Comment:
- [ ] add a check here to see if `fileName` isSimpleScript or use `getScriptType` function so this method doesn't need to be passed the `scriptType`